src/cn-batch-forms.service.js

```diff
@@ -416,6 +417,11 @@ function cnBatchForms(
           field.batchConfig.watch.push({
             resolution: `model.__batchConfig["${child.key}"] = model.__batchConfig["${field.batchConfig.key}"]`
           });
+          // if (child.key !== child._key) {
```

Comment:
I'm also not very familiar with any of this. What did this part do? Should it be removed instead of commented out, or is it needed?